src/components/courses/components/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './CourseCard.css';
+import { CourseCardProps } from './CourseCard.types';
+import Button from '../../../../common/Button';
+import { getCourseDuration } from '../../../../helpers/getCourseDuration';
+
+const CourseCard: React.FC<CourseCardProps> = ({
+  title,
+  creationDate,
+  description,
+  duration,
+  authors,
+  authorsList,
+}) => {
+  return (
+    <div className='course-card__container'>
+      <div className='course-card__description-block'>
+        <h2>{title}</h2>
+        <p className='course-card__description'>{description}</p>
+      </div>
+      <div className='course-card__creation-info-container'>
+        <p className='course-card__creation-info'>
+          <strong>Authors:</strong>{' '}
+          {authors.map((author, index) => (
```

Comment:
```suggestion
          {authors.map((authorId, index) => (
```
If I'm not mistaken.